Sindri-Labs/sindri-resources
# Correcting Gnark Version Requirements

Our database circuits for gnark v0.8.1 incorrectly listed v0.8.0 in their `go.mod` configurations.  This was silently ignored and the circuits were compiled via v0.8.1, without issue so far.  However, there are a few cases where the behavior might not match what a user is expecting.  For this reason, circuits with a non-matching gnark version between their Sindri manifest and go.mod file will cause an error going forward.  This PR brings our v8 gnark circuits up to date. 

## Testing Instructions
Submit both circuits (and their compressed versions) to prod and ensure successful compilation:
- [ ] Cubic_bls12_377
- [ ] Exponentiate
